src/guardian/keys-validation/keys-validation.spec.ts

```diff
@@ -47,95 +49,67 @@ describe('KeysValidationService', () => {
     jest.spyOn(loggerService, 'log').mockImplementation(() => undefined);
   });
 
-  it('should find and return invalid keys from the provided list', async () => {
```

Comment:
don't we need this test case?

Reply:
list of tests there revised
we still have test that validate without  use of cahce , after with use of cache and than if deposited data were changed valdiate again without cche